packages/contracts/src/dollar/mocks/MockChainLinkFeed.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import {AggregatorV3Interface} from "@chainlink/interfaces/AggregatorV3Interface.sol";
+
+contract MockChainLinkFeed is AggregatorV3Interface {
+    uint80 roundId;
+    int256 answer;
+    uint256 startedAt;
+    uint256 updatedAt;
+    uint80 answeredInRound;
+
+    constructor() {
```

Comment:
this doesn't look like it's testing the implementation as diamond facets can't have constructors, all others tests, do not have constructors, can you recheck this

Reply:
I'll check it

Reply:
Yes, this is just a mock to simulate the ChainLink data feed. It is used for unit tests and will not be deployed with diamond facets.